dbt-spark/CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# dbt-spark Changelog
+
+- This file provides a full account of all changes to `dbt-spark`.
+- Changes are listed under the (pre)release in which they first appear. Subsequent releases include changes from previous releases.
+- "Breaking changes" listed under a version may require action from end users or external maintainers when upgrading to that version.
+- Do not edit this file directly. This file is auto-generated using [changie](https://github.com/miniscruff/changie). For details on how to document a change, see [the contributing guide](https://github.com/dbt-labs/dbt-spark/blob/main/CONTRIBUTING.md#adding-changelog-entry)
+
+## Previous Releases
+For information on prior major and minor releases, see their changelogs:
+- [1.6](https://github.com/dbt-labs/dbt-spark/blob/1.6.latest/CHANGELOG.md)
```

Comment:
did we just like stop updating this :(

Reply:
Yeah, at some point the script that created the final release, or release branch, or whatever did that stopped doing that. It's a good item to fix when everything is all together.